dll/win32/winmm/registry.c

```diff
@@ -60,14 +63,16 @@ BOOL LoadRegistryMMEDrivers(char* key)
             if ( ! stricmp("wavemapper", value_name) )
             {
                 TRACE("Found a Wave-mapper: %s\n", value_data);
-                valid_driver = TRUE;
+                /* Delay loading Wave mapper driver */
+                strcpy(wavemapper, value_data);
```

Comment:
What if there are 2 wave mapper drivers, is this possible?

Reply:
Not possible, there is only one mapper for wave and one for MIDI.

Each mapper forwards the streams to one or many actual devices depending on rules in registry.